roles/sap_general_preconfigure/tasks/RedHat/assert-installation.yml

```diff
@@ -156,6 +156,23 @@
     loop_var: line_item
   ignore_errors: "{{ sap_general_preconfigure_assert_ignore_errors | d(false) }}"
 
+- name: Get install status of required IBM packages
+  ansible.builtin.shell: set -o pipefail && yum info installed {{ __sap_general_preconfigure_required_ppc64le | map('quote') | join(' ') }} | awk '/Name/{n=$NF}/Version/{v=$NF}/Release/{r=$NF}/Description/{printf ("%s\n", n)}'
+  register: __sap_general_preconfigure_register_required_ppc64le_packages_assert
+  changed_when: no
```

Comment:
Should this be checked:
`when: ansible_architecture == "ppc64le"`

Reply:
> Should this be checked: `when: ansible_architecture == "ppc64le"`
yes, thanks for this!